docs/README.md

```diff
@@ -1,6 +1,6 @@
 ## OpenFaaS Cloud installation guide
 
-Installing OpenFaaS Cloud requires some initial setup with GitHub or GitLab and a working installation of OpenFaaS. If you'd like help please see below for how to join the Slack workspace.
+Installing OpenFaaS Cloud requires some initial setup with GitHub or GitLab and a working installation of OpenFaaS accessible from the public internet, see [deployment guides](https://github.com/openfaas/faas/tree/master/guide#deployment-guides-start-here). If you'd like help please see below for how to join the Slack workspace.
```

Comment:
I think `accessible to the public internet` would be better

Reply:
Please keep the existing sentence as-is an add a new one with your content so that we don't make it too long / hard to read.

It doesn't have to be accessible from the internet - you can invoke it locally using HMAC signing in the CLI and a test event. You can also use ngrok or other tools to connect to GitHub.

Reply:
True, it would be worth adding steps in the "test it out" section for using a test event if you have the steps.